jujugui/static/gui/src/app/assets/javascripts/jujulib/juju.js

```diff
@@ -347,6 +350,9 @@ var module = module;
         }, this);
       }
       if (processed.entityType === 'bundle') {
+        if (meta['bundle-unit-count']) {
```

Comment:
I don't think that this is the correct way to protect against this, what will the component display in the event this isn't defined?